### ISSUE TITLE
Use GraalVM to produce native images

### ISSUE THREAD

**Issue by ggrossetie** [pepijnve/ditaa#12]
On https://kroki.io, I'm now using PlantUML and UMlet as native binaries. I would like to do the same with `ditaa` to make the gateway server more reliable.

- https://github.com/ggrossetie/plantuml/releases/tag/v1.2023.6
- https://github.com/yuzutech/umlet/releases/tag/2023-03-20_UMLet_v15.1

I can submit a pull request to add a GitHub Actions that will build native binaries (from a tag) and publish the binaries into  a release. What do you think?

**Comment by pepijnve** [comment]
No strong feelings either way. It's not something I would use myself, but I'm fine with letting GitHub build the images.

Just out of curiosity, how does this help with stability? Is it to keep  memory usage of the main server process in check?

**Comment by ggrossetie** [comment]
> Just out of curiosity, how does this help with stability? Is it to keep memory usage of the main server process in check?

Yes, we found a few memory leaks and infinite loops in UMlet/PlantUML/Ditaa. Since I'm using Vert.x with a relatively low number of threads it can severely impact performance/stability.
